src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -312,6 +331,14 @@ public function iShouldSeeASingleOrderInTheList(int $number = 1): void
         Assert::same($this->responseChecker->countCollectionItems($this->client->getLastResponse()), $number);
     }
 
+    /**
+     * @Then /^I should be notified that the (order|shipment) confirmation email has been successfully resent to the customer$/
```

Comment:
```suggestion
     * @Then /^I should be notified that the order confirmation email has been successfully resent to the customer$/
```
Context leak between PRs, you'll most likely have conflicts either way